homeassistant/components/sensor/canary.py

```diff
@@ -4,25 +4,36 @@
 For more details about this platform, please refer to the documentation at
 https://home-assistant.io/components/sensor.canary/
 """
+
+from canary.api import SensorType
```

Comment:
You can't import from requirements on the module level since the requirements are installed during run time. Those imports have to be done inside the functions where the imported names are needed.

Reply:
I moved this to local import